<?xml version="1.0"?>

<launch>
    <node name="richbeam_lidar" pkg="lakibeam1" type="lakibeam1_pcd_node" output="screen">
    <remap from="/richbeam_lidar/pcd" to="/pcd" />
        <param name="frame_id" type="string" value="laser"/><!--frame_id设置-->
        <param name="output_topic" type="string" value="pcd" /><!--topic设置-->
        <param name="inverted" type="bool" value="false"/><!--配置是否倒装,true倒装-->
        <param name="hostip" type="string" value="0.0.0.0"/><!--配置本机监听地址，0.0.0.0表示监听全部-->
        <param name="sensorip" type="string" value="192.168.198.2"/><!--配置sensor地址-->
        <param name="port" type="string" value="2368"/><!--配置本机监听端口-->
        <param name="angle_offset" type="int" value="0"/><!--配置点云旋转角度，可以是负数-->

        <param name="scanfreq" type="string" value="30" /><!--配置扫描频率，范围：10、20、25、30-->
        <param name="filter" type="string" value="3" /><!--配置滤波选项，范围：3、2、1、0 -->
        <param name="laser_enable" type="string" value="true" /><!--雷达扫描使能，范围：true、false-->
        <param name="scan_range_start" type="string" value="45" /><!--雷达扫描起始角度，范围：45~315-->
        <param name="scan_range_stop" type="string" value="315" /><!--雷达扫描结束角度，范围：45~315，结束角度必须大于起始角度-->
    </node>
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find lakibeam1)/rviz/lakibeam1_pcd.rviz" />
</launch>
